src/config/globals.ts

```diff
@@ -18,3 +18,12 @@ export enum YES_NO_VOTES {
 export const WRITE_IN_CANDIDATE_MAX_LENGTH = 40
 export const LOW_BATTERY_THRESHOLD = 0.25
 export const QUIT_KIOSK_IDLE_SECONDS = 5 * 60 // 5 minutes
+export const AMERICA_TIMEZONES = [
+  'America/Honolulu',
+  'America/Anchorage',
+  'America/Los_Angeles',
+  'America/Phoenix',
+  'America/Salt_Lake_City',
```

Comment:
I don't think this is a real IANA timezone. It's not in the [database](https://www.iana.org/time-zones) and [time.is says SLC's IANA identifier is America/Denver](https://time.is/Salt_Lake_City). I think we should replace this with `America/Denver`.

```suggestion
  'America/Denver',
```

Also, I think it's worth considering using different labels for these, like "Hawaiian Time", "Pacific Time", "Eastern Time", etc.

Reply:
Yes. Let's use more friendly names.